sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -1372,15 +1374,15 @@ export class BlobClient extends StorageClient {
     options.conditions = options.conditions || {};
     try {
       ensureCpkIfSpecified(options.customerProvidedKey, this.isHttps);
-      return await this.blobContext.setHTTPHeaders({
+      return await this.blobContext.setHttpHeaders({
         abortSignal: options.abortSignal,
-        blobHTTPHeaders,
+        blobHttpHeaders: blobHTTPHeaders,
         leaseAccessConditions: options.conditions,
         modifiedAccessConditions: {
           ...options.conditions,
           ifTags: options.conditions?.tagConditions
         },
-        cpkInfo: options.customerProvidedKey,
+        //cpkInfo: options.customerProvidedKey,
```

Comment:
should this be removed?

Reply:
Have we ever supported this option? If not, we may need to deprecate this option.

Reply:
This is an issue in Swagger file. In official API document, the SetHTTPHeaders operations supports CPK headers, but these headers are not defined in Swagger file, so current JS SDK doesn't have this parameter in SetHTTPHeaders operation. FE team confirmed to fix swagger file, this parameter will be added accordingly. Here I just comment it out, when the swagger file is fixed, I'll change it back.